pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -36,7 +36,7 @@ const (
 	// ErrReasonExistingAntiAffinityRulesNotMatch is used for ExistingPodsAntiAffinityRulesNotMatch predicate error.
 	ErrReasonExistingAntiAffinityRulesNotMatch = "node(s) didn't satisfy existing pods anti-affinity rules"
 	// ErrReasonAffinityNotMatch is used for MatchInterPodAffinity predicate error.
```

Comment:
Just thinking out loud, all those comment is outdated, those should be "InterPodAffinity" instead of legacy "MatchInterPodAffinity".